hands-on-exercise/1-ignite-cli/4-create-message.md

```diff
@@ -13,20 +13,20 @@ tags:
 
 Make sure you have everything you need before proceeding:
 
-* You understand the concepts of [transactions](/academy/2-cosmos-concepts/3-transactions.md), [messages](/academy/2-cosmos-concepts/4-messages.md), and [Protobuf](/academy/2-cosmos-concepts/6-protobuf.md).
-* Go is installed.
-* You have the checkers blockchain scaffold with the `StoredGame` and its helpers. If not, follow the [previous steps](./3-stored-game.md) or check out the [relevant version](https://github.com/cosmos/b9-checkers-academy-draft/tree/full-game-object).
+- You understand the concepts of [transactions](/academy/2-cosmos-concepts/3-transactions.md), [messages](/academy/2-cosmos-concepts/4-messages.md), and [Protobuf](/academy/2-cosmos-concepts/6-protobuf.md).
+- Go is installed.
+- You have the checkers blockchain scaffold with the `StoredGame` and its helpers. If not, follow the [previous steps](./3-stored-game.md) or check out the [relevant version](https://github.com/cosmos/b9-checkers-academy-draft/tree/full-game-object).
```

Comment:
No need to change this `*` to `-` on this file only. You must have a different linter.